mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -2212,16 +2340,18 @@ struct GridwiseGemmV2RewritePattern : public OpRewritePattern<miopen::GridwiseGe
 
```

Comment:
abstain from reviewing the rest of `LowerMIOpenOps.h`

Reply:
I'm not sure how much of the change in the `transform op` are actually necessary in the above> I believe we are probably able to drop some or all of the `transform op` in the future when we drop the entire cpp backend.